### PR TITLE
ADEV-46 : Fix menu hierarchy. ADEV-38 : Current page links should be highlighted

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

File: site/src/main/java/com/google/gwt/site/markdown/fs/FolderConfig.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package com.google.gwt.site.markdown.fs;
```
Comment (christiangoudreau) [inline site/src/main/java/com/google/gwt/site/markdown/fs/FolderConfig.java]: license header

**Comment by christiangoudreau** [comment]
LGTM

File: site/src/main/site/assets/less/style.less [hunk 2]
```diff
@@ -259,7 +262,7 @@ aside#submenu {
     .box-sizing();
     .transition();
 
-    &:hover {
+    &:hover, &.selected {
```
Comment (jasonlemay) [inline site/src/main/site/assets/less/style.less]: &.selected on new line

**Comment by spg** [comment]
great job, lgtm

**Comment by christiangoudreau** [comment]
LGTM

**Pull request merged**